doc/source/releases/20.01_announce_user.rst

```diff
@@ -8,20 +8,37 @@ January 2020 Galaxy Release (v 20.01)
 Highlights
 ===========================================================
 
-**Feature1**
-  Feature description.
+**Workflow Executions Menu**
+  This menu can be reached from the User menu -> Workflow Invocations
+  and lists your recent Workflow executions, their status and links
+  to the Workflow Editor and the History with the results of the
+  Workflow run.
 
-**Feature2**
-  Feature description.
+.. figure:: https://user-images.githubusercontent.com/216771/65520097-0a041f80-deb5-11e9-80ac-2557b36c252f.png
+   :alt: Workflow invocation menu
 
-**Feature3**
-  Feature description.
+.. figure:: https://user-images.githubusercontent.com/6804901/73737075-345ac900-4742-11ea-9f50-0f0e4c3d9271.png
+   :alt: Workflow invocatios
+
+**Galay Markdown Pages and Workflow Reports as PDF**
+  An additional link has been added to Galaxy Markdown Pahes and Workflow reports
+  that exports your document as a standalone PDF.
```

Comment:
Maybe a picture of the report as PDF? How does it look when it's converted, if there's a galaxy output embedded or so?